### PR TITLE
Update hu.json

### PR DESCRIPTION
**Pull request by sharkmu**
This PR translates some strings that aren't translated yet.

### CODE REVIEW

File: hu.json [hunk 1]
```diff
@@ -280,15 +280,15 @@
 			"option-catalyst-desc": "Jelenleg {{tier}} Katalizátor licenc rendelkezésére áll. Köszönjük a támogatását!",
 			"option-catalyst-desc-no-license": "Jelenleg nincs Katalizátor licenc. A Katalizátor licenc egyszeri adomány az Obsidian fejlesztésének támogatására.",
 			"label-app": "Alkalmazás",
-			"label-add-own-language": "See how to add your language to Obsidian."
+			"label-add-own-language": "Nézze meg, hogy hogyan lehet hozzáadni a saját nyelvét az Obsidianhoz."
```
Comment (joethei) [inline hu.json:283]: Please don't translate the word "Obsidian".
It's a brand name and should not be translated.

**Review by joethei — CHANGES_REQUESTED** [review]

File: hu.json (hunk 1 above)
Comment (sharkmu) [reply hu.json:283]: I didn't translate it. I just added affix to it

**Review by joethei — APPROVED** [review]

**Comment by joethei** [comment]
Hi @sharkmu sorry for letting you wait.
Could you please fix the merge conflicts?

**Comment by sharkmu** [comment]
@joethei Hey! No problem. How about now?

**Comment by joethei** [comment]
There are still two merge conflicts.

**Comment by joethei** [comment]
Closing as this PR has gone stale.
If you are ready continue feel free to reopen it.

**Pull request closed without merging**